src/test/java/com/dropwizard/example/integration/IntegrationTest.java

```diff
@@ -0,0 +1,148 @@
+package com.dropwizard.example.integration;
+
+import com.dropwizard.example.SimpleApplication;
+import com.dropwizard.example.model.Person;
+import config.configuration.SimpleConfiguration;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.*;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+import static org.junit.Assert.assertEquals;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class IntegrationTest {
+
+    private static Client client;
+    private static String CONFIG_PATH = "src/test/resources/config.yml" ;
+
+    @ClassRule
+    private static DropwizardAppExtension<SimpleConfiguration> EXT = new DropwizardAppExtension<>(
+            SimpleApplication.class, CONFIG_PATH);
+
+    @BeforeAll
+    public static void migrateDb() throws IOException {
+        client = EXT.client();
+
+    }
+
+    @AfterAll
+    public static void tearDown() throws IOException {
+        client.close();
+    }
+
+    @Test
+    void testGetPersonSuccess() {
+        // try to get person with ID=1
+        final Person person = new Person(1L, "Dan", "Johnson");
```

Comment:
Why you need create this person object? Just declare it's properties (id, first name, last name).